src/parquet/properties.h

```diff
@@ -38,6 +39,161 @@ struct ParquetVersion {
 static int64_t DEFAULT_BUFFER_SIZE = 0;
 static bool DEFAULT_USE_BUFFERED_STREAM = false;
 
+// should find a better name???
```

Comment:
In the Java version, I'm using the following API names: FileEncryptionProperties, FileDecryptionProperties (encryption and decryption require different parameters), and ColumnEncryptionProperties (no ColumnDecryptionProperties, since these are reduced to a single parameter only, an explicit key, doesnt justify a class addition). Lets try to keep the API names similar in both Java and C++ versions. We'll get to the bottom of this later, when working on the interoperability.